compiler/semexprs.nim

```diff
@@ -707,19 +707,17 @@ proc evalAtCompileTime(c: PContext, n: PNode): PNode =
           break maybeLabelAsStatic
       n.typ = newTypeWithSons(c, tyStatic, @[n.typ])
       n.typ.flags.incl tfUnresolved
-
```

Comment:
Don't rewrite logic to suit your style, now the fix is unreadable. Is the fix as simple as ``callee.kind notin {skProc, skFunc, skConverter, skConst}``? Who knows.

Reply:
Ok I'll revert it and just add the relevant bit back. Just thought I'd clean up since I was there.